docsify/sidebar.md

```diff
@@ -0,0 +1,22 @@
+* [Readme](/)
+* [Comparaison](/docs/comparison.md)
```

Comment:
here it's `comparisons.md`

Reply:
Good catch, it's fixed in the latest commit.